src/levels/level-A.js

```diff
@@ -82,11 +82,16 @@ const LevelA = ({ squares, size, isStarted, isRestart, setRestartComplete }) =>
   const [begin, setBegin] = useState(false)
 
   useEffect(() => {
-    if (checkIfComplete(currentSquares)) {
-      setIsComplete(true)
+    const completeStatus = checkIfComplete(currentSquares);
+    console.log(completeStatus);
```

Comment:
please make the changes I requested and change back to the original logic with the additional checks for isStarted and the restart variable; if you do this, you will go back to console logging "isComplete" which means you don't need that additional use effect as this variable will be in use. if you face any other build issues, lmk. Please check when running locally that there are no warnings, as this causes the build to fail